pypxe-server.py

```diff
@@ -18,7 +18,7 @@
 JSON_CONFIG = ''
 
 #Default Network Boot File Directory
-NETBOOT_DIR = 'netboot'
+NETBOOT_DIR = os.path.abspath(os.path.join(os.getcwd(), 'netboot'))
```

Comment:
`os.path.abspath` has an implicit `os.getcwd`, so `os.path.abspath('netboot')` is functionally identical.